packages/compass-indexes/src/components/create-index-actions/create-index-actions.tsx

```diff
@@ -24,16 +24,16 @@ const createIndexButtonStyles = css({
  */
 function CreateIndexActions({
   error,
-  clearError,
+  errorCleared,
   inProgress,
   createIndex,
-  closeCreateIndexModal,
+  createIndexClosed,
```

Comment:
Very much a nit, but I'll continue mentioning it everywhere I see it 😄 While redux convection for action names is to "describe events that occurred", in React world callback props are usually named after UI actions that trigger those props with an `on` prefix: so instead of `errorCleared`, you might want the prop to be `onErrorBannerClose` (or `onErrorBannerCloseClick`) instead, this just better describes what component is doing, the actual logic of what happens on this user interaction can change, but interaction will stay the same

Reply:
Yeah it is on my TODO list to name these properly. That's just the initial result of find/replace.

Plan is to onErrorCleared for the prop, errorCleared for a plain action creator that just describes an event that occurred and clearError if it happens to be a thunk dispatch (which in turn would result in action creators that just describe events).